src/synapse/azext_synapse/_params.py

```diff
@@ -0,0 +1,161 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long
+from azure.cli.core.commands.parameters import name_type, tags_type, get_three_state_flag, get_enum_type, \
+    resource_group_name_type
+
+
+# pylint: disable=too-many-statements
+def load_arguments(self, _):
+    # synapse spark
+
+    for scope in ['batch', 'session', 'session-statement']:
+        with self.argument_context('synapse spark ' + scope) as c:
+            c.argument('workspace_name', help='The name of the workspace.')
+            c.argument('spark_pool_name', help='The name of the spark pool.')
+
+    for scope in ['synapse spark batch', 'synapse spark session']:
+        with self.argument_context(scope + ' create') as c:
+            c.argument('job_name', arg_type=name_type, help='The spark batch or session job name.')
+            c.argument('file', help='The URI of file.')
+            c.argument('class_name', help='The class name.')
+            c.argument('args', nargs='+', help='The arguments of the job.')
+            c.argument('jars', nargs='+', help='The array of jar files.')
+            c.argument('files', nargs='+', help='The array of files URI.')
+            c.argument('archives', nargs='+', help='The array of archives.')
+            c.argument('conf', help='The configuration of spark batch job.')

```

Comment:
>conf [](start = 24, length = 4)

config

Reply:
This is livy standard parameter, our customer is used to conf instead of config